### PR TITLE
Hosting Helm chart repository in GitHub Pages.

### PR DESCRIPTION
**Pull request by karok2m**
Changes:
* Created Helm chart repository that is hosted in out GitHub Pages.
* Updated `docs/deploy/howto-deploy-helm.md` with details about consuming from new Helm chart repository.
* Updated `docs/dev-guides/howto-release-helm-chart.md` with details about publishing to new Helm chart repository.
* Minor fix in `docs/manual/readme.md`.

### CODE REVIEW

File: docs/dev-guides/howto-release-helm-chart.md [hunk 1]
```diff
@@ -93,9 +94,34 @@ helm repo index . --url https://azureiiot.blob.core.windows.net/helm/ --merge '<
 
 Documentation of `helm repo index` command can be found [here](https://helm.sh/docs/helm/helm_repo_index/).
 
-Now `azure-industrial-iot-0.4.0.tgz` file and updated `index.yaml` file can be uploaded to `helm` container of
+Now `azure-industrial-iot-0.4.0.tgz` and updated `index.yaml` files can be uploaded to `helm` container of
```
Comment (cristipogacean) [inline docs/dev-guides/howto-release-helm-chart.md]: why not azure-industrial-iot-0.4.1.tgz? isn't this the latest release?

File: docs/dev-guides/howto-release-helm-chart.md [hunk 2]
```diff
@@ -8,6 +8,7 @@
   * [Publish the Helm Chart](#publish-the-helm-chart)
     * [Create a Helm Chart Package](#create-a-helm-chart-package)
     * [Publish to `https://azureiiot.blob.core.windows.net/helm` Repository](#publish-to-httpsazureiiotblobcorewindowsnethelm-repository)
```
Comment (cristipogacean) [inline docs/dev-guides/howto-release-helm-chart.md:10]: is this still valid?

File: docs/dev-guides/howto-release-helm-chart.md (hunk 1 above)
Comment (karok2m) [reply docs/dev-guides/howto-release-helm-chart.md]: This is a dev guide so I didn't want to change all the numbers there. I can change them as well.

**Comment by karok2m** [comment]
Let's add a recommendation to use the one from GitHub Pages in our docs.

File: docs/dev-guides/howto-release-helm-chart.md (hunk 2 above)
Comment (karok2m) [reply docs/dev-guides/howto-release-helm-chart.md:10]: Yes, we still have the repo in Storage Account under same URL.

**Review by marcschier — DISMISSED** [review]

File: docs/dev-guides/howto-release-helm-chart.md (hunk 1 above)
Comment (karok2m) [reply docs/dev-guides/howto-release-helm-chart.md]: Updated values for `2.8.2` release. Added note about version part in `tgz` file name.

**Comment by karok2m** [comment]
> Let's add a recommendation to use the one from GitHub Pages in our docs.

Added recommendation to use new Helm chart repository.

**Review by cristipogacean — APPROVED** [review]

**Pull request merged**